benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -5,8 +5,8 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Report-Msgid-Bugs-To: https://github.com/cal-itp/benefits/issues \n"
-"POT-Creation-Date: 2023-01-26 16:08+0000\n"
+"Report-Msgid-Bugs-To: \n"
```

Comment:
Why did we lose our Issues link here?

Reply:
It looks like the helper `bin/makemessages.sh` was not run, it is supposed to put this line back.